lib/matplotlib/cbook.py

```diff
@@ -2445,6 +2445,14 @@ def safe_first_element(obj):
     return next(iter(obj))
 
 
+def asSequence(data):
```

Comment:
How about ensure_sequence ?

On Thu, Jul 21, 2016 at 9:45 PM Eric Firing <notifications@github.com>
wrote:

> In lib/matplotlib/cbook.py
> <https://github.com/matplotlib/matplotlib/pull/6787#discussion_r71816398>:
>
> > @@ -2445,6 +2445,14 @@ def safe_first_element(obj):
> >      return next(iter(obj))
> >
> >
> > +def asSequence(data):
>
> Naming is hard. A problem here is that where this is used, there is no
> clue that the "view" in question is the obscure "MappingView", not a "view"
> in the numpy sense, which is much more common. "convert_dict_view"?
> "convert_MappingView"?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/6787/files/efc53d5d021a853757495099716d6c34166179b0#r71816398>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAMMhVT7zHoSEmkgfeLd_V8rlePZDhzgks5qYCCcgaJpZM4JOX_W>
> .
>